examples/prediction_upload/geospatial_predictions.ipynb

```diff
@@ -0,0 +1,1563 @@
+{
```

Comment:
<p>Please remove the api key, I also recommend to delete this key from your account.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Labelbox/labelbox-python/pull/867/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>